cobaya/model.py

```diff
@@ -551,6 +551,12 @@ def _tidy_requirements(_component, _require):
 
             for k in can_provide + component.output_params + provide_params:
                 providers[k] = providers.get(k, []) + [component]
+        # Add requirements requested by hand
+        _manual = '_manual'
```

Comment:
Could just use None rather than '_manual'? Looks odd mixing strings and Components